src/main/java/org/openhab/tools/analysis/checkstyle/AboutHtmlCheck.java

```diff
@@ -62,19 +61,16 @@ public void setValidAboutHtmlFileURL(String validAboutHtmlFileURL) {
 
     @Override
     public void beginProcessing(String charset) {
-        InputStream validAboutHtmlInputStream = null;
+        CachingHttpClient<String> cachingClient = new CachingHttpClient<>(data -> new String(data));
```

Comment:
I think that this implementation will change it to a download for each binding, I would be surprised if this already works over all the bindings?

Reply:
Actually it works :). The cache is a static field inside the client.

Reply:
Then its great 😅